dub.json

```diff
@@ -7,6 +7,7 @@
 	"dependencies": {
         "derelict-cl"  : "~>2.0.0",
         "derelict-cuda": "~>2.0.1",
+        "taggedalgebraic": "~>0.10.7"
```

Comment:
I'll be using it to bridge the different OpenCL versions and CUDA versions.
Only way to do it without resorting to classes (which won't work so well with templates.